Client/UIDeviceExtensions.swift

```diff
@@ -0,0 +1,21 @@
+//
+// Copyright (c) 2017-2019 Cliqz GmbH. All rights reserved.
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+//
+
+import UIKit
+
+extension UIDevice {
```

Comment:
I was confused why you created this extension at first, because I thought you can use traitCollection everywhere, but after reading through your code, I think you're right.